src/ui.tsx

```diff
@@ -750,6 +714,237 @@ class UserInterface extends Component<Props, State> {
       </MuiCard>
     );
 
+    const defaultColumns = [
+      {
+        headerName: "Image Name",
+        field: "imageName",
+        width: 150,
+        editable: false,
+      },
+      {
+        headerName: "Annotation Progress",
+        field: "annotationProgress",
+        width: 150,
+        editable: false,
+      },
+      {
+        headerName: "Assignees",
+        field: "assignees",
+        width: 150,
+        editable: false,
+      },
+      {
+        headerName: "Labels",
+        field: "labels",
+        width: 250,
+        editable: false,
+        renderCell: (params: GridRenderCellParams<any, any, any>) => {
```

Comment:
```suggestion
renderCell: (params: GridRenderCellParams<unknown, MetaItem, unknown>) => {
```

should remove the need to ignore ES Lint below

Reply:
and also you might not need to cast to `string[]` on the line below